test/integration/partials-test.js

```diff
@@ -0,0 +1,41 @@
+const assert = require('assert');
+const path = require('path');
+const dist = path.resolve(__dirname, '..', '..', 'dist');
+
+describe('Service worker partials', function() {
```

Comment:
Should these test check all of the expected values for the partials, as opposed to just checking the RSS field?

In other words, should we use this file as the general integration test for the entire service work partial.